tasks/test.py

```diff
@@ -592,7 +592,7 @@ def install_shellcheck(ctx, version="0.7.0", destination="/usr/local/bin"):
         platform = "linux"
 
     ctx.run(
-        "wget -qO- \"https://storage.googleapis.com/shellcheck/shellcheck-v{sc_version}.{platform}.x86_64.tar.xz\" | tar -xJv -C /tmp".format(
+        "wget -qO- \"https://github.com/koalaman/shellcheck/releases/download/v{sc_version}/shellcheck-v{sc_version}.{platform}.x86_64.tar.xz\" | tar -xJv -C /tmp".format(
```

Comment:
Nit: for next time this should be in a different PR.

Reply:
It's just cherrypick from master because circleci was failing.

Reply:
In this case my advice would be to:
```
# assuming we're in your-branch
git checkout master && git pull
git checkout your-branch
git rebase master
```

or alternatively:
```
# assuming we're in your-branch
git merge origin master
```

Reply:
Yes, `rebase` is your best friend.